File: readme/cn/selfhost/SDK/webfont-SDK-java(for1.22.2)/cn/webfont/client/model/WebFontFace.java
package cn.webfont.client.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.core.JsonProcessingException;
import com.fasterxml.jackson.databind.ObjectMapper;
import com.fasterxml.jackson.databind.SerializationFeature;
import lombok.Data;

import java.io.Serializable;

@Data
public class WebFontFace implements Serializable {
  @JsonProperty("FontFamily")
  private String fontFamily = "";

  @JsonProperty("FontFace")
  private String fontFace = "";

  @JsonProperty("Tag")
  private String tag = "";

  @JsonProperty("Code")
  private int code = 200;

  @JsonProperty("ErrorMessage")
  private String errorMessage = "";

  @JsonProperty("Woff")
  private String woff = "";

  public String toJson() throws JsonProcessingException {
    // 以json格式输出
    ObjectMapper mapper = new ObjectMapper();
    mapper.configure(SerializationFeature.INDENT_OUTPUT, true);
    return mapper.writeValueAsString(this);
  }
}

File: readme/cn/selfhost/SDK/webfont-SDK-NetCore(for1.22.2)/WebFontClient.SDK/Models/WebFontFace.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Text;

namespace WebFontClient.SDK.Models
{
    public class WebFontFace
    {
        public string FontFamily { get; set; } = "";

        public string FontFace { get; set; } = "";

        public string Tag { get; set; } = "";

        public int Code { get; set; } = 200;

        public string ErrorMessage { get; set; } = "";

        public string Woff { get; set; } = "";

        /// <summary>
        /// 输出此类的JSON格式
        /// </summary>
        /// <returns></returns>
        public string ToJson()
        {
            return JsonConvert.SerializeObject(this);
        }
    }
}
